### ISSUE TITLE
Support user collection pages

### ISSUE THREAD

**Issue by Otiel** [Otiel/BandcampDownloader#87]
Support downloading albums from a Bandcamp user (≠ artist) collection: `https://bandcamp.com/[user]`

**Comment by pjfuller1976** [comment]
Will this feature be implemented soon? I have over 800 albums, it would be much easier to paste in your bandcamp user site, then each album. I initially thought it would be a few clicks to download everything you have.

**Comment by Otiel** [comment]
No, sorry, I won't implement this in the foreseeable future. Not enough time. 😞
